src/components-examples/material/stepper/stepper-intl/stepper-intl-example.html

```diff
@@ -0,0 +1,62 @@
+<label>Pick the text for the optional label</label>
+<mat-radio-group
+  aria-label="Select an option for the optional label"
+  class="demo-radio-group"
+  [(ngModel)]="optionalLabelText"
+  (ngModelChange)="updateOptionalLabel()"
+>
```

Comment:
nit: Move the `>` up to the end of `(ngModelChange)="updateOptionalLabel()"` (do this in all places)